addons/project/views/project_views.xml

```diff
@@ -25,13 +25,16 @@
                     <field name="parent_id"/>
                     <field name="rating_last_text"/>
                     <filter string="My Tasks" name="my_tasks" domain="[('user_ids', 'in', uid)]"/>
+                    <filter string="Followed Tasks" name="followed_by_me" domain="[('message_is_follower', '=', True)]"/>
                     <filter string="Unassigned" name="unassigned" domain="[('user_ids', '=', False)]"/>
                     <separator/>
                     <filter string="Starred" name="starred" domain="[('priority', 'in', [0, 1])]"/>
+                    <filter string="Not Starred" name="not_starred" domain="[('priority', 'not in', [0, 1])]"/>
```

Comment:
Since there is only 2 tuples in this Selection field, how do `Starred` and `Not Starred` work ? 🤔 
Did I miss something ?

Reply:
@xavierbol yeah, Here we need to change both filter's domains. Thanks 🙂